tsconfig.json

```diff
@@ -0,0 +1,16 @@
+{
+  "include": ["lib/**/*.js", "test/**/*.js"],
+  "compilerOptions": {
+    "target": "ES2020",
+    "lib": ["ES2020"],
+    "module": "ES2020",
+    "moduleResolution": "node",
+    "allowJs": true,
+    "checkJs": true,
+    "declaration": true,
+    "emitDeclarationOnly": true,
+    "allowSyntheticDefaultImports": true,
```

Comment:
Is this needed?
Modules that have native ESM generally don't, unless there are still some CJS only dependencies.

Reply:
I think this field was for cjs in esm as node supports it. I haven't really gave it much thought, but it's this way in all the repo's I've updated 🤔